src/web.c

```diff
@@ -48,8 +48,11 @@ const char *xml =
 	" <device>\r\n"
 	"  <deviceType>urn:schemas-upnp-org:device:InternetGatewayDevice:1</deviceType>\r\n"
 	"  <friendlyName>%s</friendlyName>\r\n"
-	"  <manufacturer>%s</manufacturer>\r\n%s"
+	"  <manufacturer>%s</manufacturer>\r\n"
+	"  <manufacturerURL>%s</manufacturerURL>"
```

Comment:
This breaks backwards compatibility, the default value of mrfurl is taken from the configure script **as XML**.  That's why the line looks so "odd":

    "  <manufacturer>%s</manufacturer>\r\n%s"

the last `%s` is for inserting the XML.

Also, the `manufacturerURL` tag cannot be empty, so if `mrfurl` is set the XML tag must be composed separately.